draft-ietf-rtcweb-jsep.xml

```diff
@@ -741,66 +741,82 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-          receiver wants to receive non-square pixels, this cannot be
-          satisfied and the sender MUST NOT transmit the track.</t>
-
-          <t>In the special case of receiving a maximum resolution of
-          [0, 0], as described above, the sender MUST NOT transmit the
-          track.</t>
+          <t>This specification prescribes more specific behavior.
+          When a MediaStreamTrack, which is producing
+          video of a certain resolution (the "track resolution"), is attached
+          to a RtpSender, which is encoding the track video
+          at the same or lower resolution(s) (the "encoder resolutions"),
+          and a remote description is applied that references the sender and
+          contains valid "a=imageattr recv" attributes, it MUST
+          follow the rules below to ensure the sender does not transmit a
+          resolution that would exceed the size criteria specified in the
+          attributes. These rules MUST be followed
+          as long as the attributes remain present in the remote description,
+          including cases in which the track changes its resolution,
+          or is replaced with a different track.
+          </t>
+
+          <t>Depending on how the RtpSender is configured, it may be producing
+          a single encoding at a certain resolution, or, if
+          simulcast <xref target="sec.simulcast" /> has been negotiated,
+          multiple encodings, each at their own specific resolution. In
+          addition, depending on the configuration, each encoding may have the
+          flexibility to reduce resolution when needed, or may be locked to
+          a specific output resolution.
+          </t>
+
+          <t>For each encoding being produced by the RtpSender, the following
+          rules are applied to determine what should be transmitted:
+            <list style="symbols">
+              <t>First, the right "a=imageattr recv" attribute is selected.
+              This is performed by taking the
+              attribute with the highest "q=" value from the set of attributes
+              that reference the media format that has been selected for the
+              specified encoding. If multiple attributes have the same "q="
+              value, the one that appears first in the m= section is used.
+              Note that while JSEP endpoints will include at most
+              one "a=imageattr recv" attribute per media format, JSEP
+              endpoints may receive session descriptions from non-JSEP
+              endpoints with m= sections that contain multiple such attributes.
+              </t>
+
+              <t>If there is an applicable "a=imageattr recv" attribute for
+              the encoding, the limits from the attribute are then compared to
+              the encoder resolution. Only the specific limits mentioned below
+              are considered; any other values, such as picture aspect ratio,
+              MUST be ignored.
```

Comment:
@juberti those slides does not indicate that it was inteded to be required to ignore the rest of the attributes. I think there is a big difference between mandating ignoring, and allowing for ignoring. Are there issues you see with an implementation actually taken the parameters into account?